controllers/user.js

```diff
@@ -354,7 +354,7 @@ async function getBreachStats(req, res) {
     });
   }
   const fxaResponse = await FXA.verifyOAuthToken(req.token, FXA_MONITOR_SCOPE);
-  if (fxaResponse.name === "HTTPError") {
+  if (fxaResponse.name === "HTTPError" || !fxaResponse.body.scope.includes(FXA_MONITOR_SCOPE)) {
     return res.status(fxaResponse.statusCode).json({
```

Comment:
In the `FXA_MONITOR_SCOPE` case, this is going to return a `200` response with an error message; you'll need to special-case it to return a `401` or the client will get confused.